equetzal/huronOS-build-tools
# [Installer] Replace EXTLINUX with huronOS fork

## Description
This PR updates the binaries for EXTLINUX to match the code of the PR https://github.com/huronOS/syslinux/pull/1, enabling the installer to be independent of other installations happening in parallel. 

## How to test
Checkout at the branch, install multiple instances of huronOS in parallel, check that any of them is blocked by the others. (Before all of them finished almost at the same time)

## Results
I've been able to test installations with times between 12 - 15 minutes for USB A 2.0 drives, and 8 - 12 minutes for USB C 3.0+ drives.

Solves: #84 and #87
